composer.json

```diff
@@ -41,6 +47,10 @@
         "cs-fix": "phpcbf",
         "test": "phpunit"
     },
+    "config": {
+        "sort-packages": true,
+        "bin-dir": "bin"
```

Comment:
Why changing that?

Reply:
Because I prefer like this but if you want I can remove it ;)

Reply:
Yes, this is not a useful change :p.